Makefile

```diff
@@ -206,12 +206,15 @@ deploy-ocm-secret: deploy-namespace
 	python3 ./tools/deploy_sso_secret.py --secret "$(OCM_CLIENT_SECRET)" --id "$(OCM_CLIENT_ID)" --namespace "$(NAMESPACE)" \
 		--profile "$(PROFILE)" --target "$(TARGET)" --apply-manifest $(APPLY_MANIFEST)
 
+deploy-local-auth-secret: deploy-namespace
```

Comment:
Could we avoid another Makefile target? I would prefer if it could be as part of `deploy_assisted_installer_configmap`